notebooks/403-action-recognition-webcam/403-action-recognition-webcam.ipynb

```diff
@@ -0,0 +1,691 @@
+    "                processing_times.append(stop_time - start_time)\n",
+    "                # Create a text template to show inference results over video\n",
+    "                text_template = '{label},{conf:.2f}%,{Time:.1f}ms,{fps:.1f}FPS'\n",
+    "                # Use processing times from last 200 frames\n",
+    "                if len(processing_times) > 200:\n",
+    "                    processing_times.popleft()\n",
+    "                    \n",
+    "                _, f_width = frame.shape[:2]\n",
+    "                # Mean processing time [ms]\n",
+    "                processing_time = np.mean(processing_times) * 1000\n",
+    "                fps = 1000 / processing_time\n",
+    "\n",
+    "                # Visualize the results\n",
+    "                for i in range(0, 3):\n",
+    "                    display_text = text_template.format(label=decoded_labels[i], conf=decoded_top_probs[i] * 100, Time=processing_time, fps=fps)\n",
+    "                    text_loc = (TEXT_LEFT_MARGIN, TEXT_VERTICAL_INTERVAL * (i + 1))\n",
+    "                    cv2.putText(frame, display_text, text_loc, FONT_STYLE, FONT_SIZE, FONT_COLOR)\n",
+    "                \n",
+    "                # Use this workaround if there is flickering\n",
+    "                if use_popup:\n",
+    "                    cv2.imshow(title, frame)\n",
+    "                    key = cv2.waitKey(1)\n",
+    "                    # escape = 27\n",
+    "                    if key == 27:\n",
+    "                        break\n",
+    "                else:\n",
+    "                    # Encode numpy array to jpg\n",
+    "                    _, encoded_img = cv2.imencode(\".jpg\", frame, params=[cv2.IMWRITE_JPEG_QUALITY, 90])\n",
+    "                    # Create IPython image\n",
+    "                    i = display.Image(data=encoded_img)\n",
+    "                    # Display the image in this notebook\n",
+    "                    display.clear_output(wait=True)\n",
+    "                    display.display(i)\n",
+    "    # ctrl-c\n",
+    "    except KeyboardInterrupt:\n",
+    "        print(\"Interrupted\")\n",
+    "    # Any different error\n",
+    "    except RuntimeError as e:\n",
+    "        print(e)\n",
+    "\n",
+    "    # Stop capturing\n",
+    "    player.stop()\n",
+    "    if use_popup:\n",
+    "        cv2.destroyAllWindows()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Run Action Recognition using your webcam\n",
+    "\n",
+    "> NOTE: _To use the webcam, you must run this Jupyter notebook on a computer with a webcam. If you run on a server, the webcam will not work. However, you can still do inference on a video in the final step._"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "run_action_recognition(source=0, flip=False, use_popup=False, skip_first_frames=0)"
```

Comment:
You can remove skip_first_frames. It's needed only in 402. You can check, there is no that in 401.